types/p2p.yaml

```diff
@@ -34,6 +34,12 @@ MetaData:
         - description: "Bitvector representing the node's persistent attestation subnet subscriptions."
         - $ref: "./primitive.yaml#/Hex"
         - example: "0x0000000000000000"
+    syncnets:
+      allOf:
+        - description: "Bitvector representing the node's sync committee subnet subscriptions."
```

Comment:
Should we add notice that it will be present after altair fork is activated?

Reply:
I wasn't sure of the wording, but did make an update. Happy to change it further if there's a better way of expressing it.